### PR TITLE
Add QSR demo

### PR DESCRIPTION
**Pull request by FlorianJacta**
Adding the QSR demo:

[QSR Sales Forecasting - Taipy.zip](https://github.com/Avaiga/taipy-doc/files/15110913/QSR.Sales.Forecasting.-.Taipy.zip)

image
image

image
image
image
image
image

### CODE REVIEW

File: docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+counter.
+
+[Try it live](https://qsr-fcst.taipy.cloud/data){: .tp-btn target='blank' }
+[Get it on GitHub](https://github.com/Avaiga/demo-qsr-sales-forecasting-dashboard){: .tp-btn .tp-btn--accent target='blank' }
+
+# Understanding the Application
+
+This application showcases various capabilities of Taipy, including:
+
+1. The use of Taipy Core to manage the forecasting process -- encapsulating each forecast
```
Comment (jrobinAV) [inline docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md]:
```suggestion
1. The use of Taipy Scenario management to handle the forecasting process -- encapsulating each forecast 
```

File: docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+This application showcases various capabilities of Taipy, including:
+
+1. The use of Taipy Core to manage the forecasting process -- encapsulating each forecast 
+execution as a *Scenario*, persisting the parameters and results for future reference and 
+evaluation.
+2. The use of Taipy GUI to easily create a multi-page application, featuring a navigation 
+sidebar and a top panel for selecting a *Scenario*, where both panels are present and 
+used on every page.
+3. Demonstrates a mini version of a real implemented system that uses Taipy to to
```
Comment (jrobinAV) [inline docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md]:
```suggestion
3. Demonstrates a mini version of a real implemented system that uses Taipy to 
```

File: docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+button to *submit* the scenario for execution, during which a progress bar will indicate 
+its progress.
+
+![Create Page](images/scenario_page.png){width=90% : .tp-image-border }
+
+
+## Data and Forecast Page
+
+The page for viewing the forecasts of the selected scenario, which is selected using the 
+top panel. Includes a line chart comparing the actual and forecasted sales, and a
```
Comment (jrobinAV) [inline docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md:52]: I would write 'an histogram' and not 'a histogram'. No?

**Review by jrobinAV — COMMENTED** [review]
Minor comments

File: docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/gallery/decision_support/3a_qsr_sales_forecasting/index.md:52]: It is 'a histogram'

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**